src/bun.zig

```diff
@@ -3071,13 +3096,16 @@ pub fn SliceIterator(comptime T: type) type {
 
 pub const Futex = @import("./futex.zig");
 
+pub const crash_handler = @import("crash_handler.zig");
```

Comment:
nit: feels like this should be `CrashHandler`, silimar to `bun.Global`

Reply:
i used snake case because the crash_handler does not contain any fields, similar to namespaces like `windows`, `path`, `sys` (and, the zig standard library does this naming practice too). but we have many exceptions, like Global and Output.